help/actions/_posts/2015-02-09-export.md

```diff
@@ -6,7 +6,51 @@ order: 3
 
 ### Export Formats
 
-Use this button to export your search results in different formats.  Currently available options are ADS Classic, BibTeX, AASTeX and Endnote.
+Use this button to export your search results in different formats.  Currently available options are:
+###### 1. Tagged Formats:
+* BibTeX
+* BibTeX with abstracts
+* Generic tagged abstracts
+* EndNote format
+* ProCite format
+* Refman format
+* RefWorks format
+* MEDLARS format
+
+###### 2. Latex Formats:
+* AASTeX format
+* Icarus format
+* MNRAS format
+* SoPh format
+
+###### 3. xml Formats:
+* Dublin Core XML
+* XML
+* XML with abstracts
+* VOTables
+* RSS
+
+### The Bibtex Format Configration
+
+For the two bibtex formats there are two optional parameters `maxauthor` and `keyformat` which can be specified in user preferences.
+
+`maxauthor` is maxinum number of authors displayed if number of authors exceed 200. The default values for `maxauthor` for *Bibtex* and *Bibtex with abstracts* are respectively 10 and 0, where 0 means display all authors.
+
+`keyformat` is mnemonic key for identifying a bibtex entry, default is bibcode. Key can be specified to contain the first author's last name, or some combination of publication year, journal, or additional authors. The key generation algorithm can be specified as a custom format. For example, the default bibtex export for `2019AAS...23338108A` is:
+```
+@INPROCEEDINGS{2019AAS...23338108A,
+       author = {{Accomazzi}, Alberto and {Kurtz}, Michael J. and {Henneken}, Edwin and
+...
+}
+```
```

Comment:
This citation broke the page rendering because of the curly braces. This is what I used to fix it and get the page to render (e.g. just wrap the whole thing, backticks included, in {% raw %} {% endraw %}):

{% raw %}
```
@INPROCEEDINGS{2019AAS...23338108A,
       author = {{Accomazzi}, Alberto and {Kurtz}, Michael J. and {Henneken}, Edwin and
...
}
```
{% endraw %}

Reply:
so sorry. thank you so much.